### PR TITLE
Add "your first test" section to xk6-disruptor

### PR DESCRIPTION
**Pull request by pablochacin**
Existing documentation does not offer a clear introduction to the first steps for starting using xk6-disruptor

Signed-off-by: Pablo Chacin <pablochacin@gmail.com>

### CODE REVIEW

File: src/data/markdown/docs/40 xk6-disruptor/01 Get started/01 Welcome.md [hunk 1]
```diff
@@ -33,3 +33,11 @@ xk6-disruptor is in the alpha phase, undergoing active development. k6 doesn't g
 Until this extension reaches v1.0 release, your k6 scripts may need to be updated on each release .
 
 </Blockquote>
+
+## Your first test with xk6-disruptor
+
+1. Check the [requirements](/javascript-api/xk6-disruptor/get-started/requirements)
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/40 xk6-disruptor/01 Get started/01 Welcome.md:39]:
```suggestion
1. Check the [requirements](/javascript-api/xk6-disruptor/get-started/requirements).
```

**Review by MattDodsonEnglish — COMMENTED** [review]
Looks fine, but will the user not have to run the step in "Expose your service" first?

**Comment by pablochacin** [comment]
> Looks fine, but will the user not have to run the step in "Expose your service" first?

The short answer is "it depends".  And the long answer is "it is complicated". That is explained in the example, so I don't think we should put as a step.

File: src/data/markdown/docs/40 xk6-disruptor/01 Get started/01 Welcome.md [hunk 2]
```diff
@@ -33,3 +33,11 @@ xk6-disruptor is in the alpha phase, undergoing active development. k6 doesn't g
 Until this extension reaches v1.0 release, your k6 scripts may need to be updated on each release .
 
 </Blockquote>
+
+## Your first test with xk6-disruptor
+
+1. Check the [requirements](/javascript-api/xk6-disruptor/get-started/requirements).
+
+2. [Install](/javascript-api/xk6-disruptor/get-started/installation) xk6-disrutor binary in your system.
```
Comment (MattDodsonEnglish) [inline src/data/markdown/docs/40 xk6-disruptor/01 Get started/01 Welcome.md:41]:
```suggestion
2. [Install](/javascript-api/xk6-disruptor/get-started/installation) the xk6-disrutor binary in your system.
```

**Review by MattDodsonEnglish — APPROVED** [review]

**Comment by MattDodsonEnglish** [comment]
> > Looks fine, but will the user not have to run the step in "Expose your service" first?
> 
> The short answer is "it depends". And the long answer is "it is complicated". That is explained in the example, so I don't think we should put as a step at this level.
> 
> Also, notice I'll be iterating on the example(s) to simplify them and remove the friction points like this, but unfortunately there's some intrinsic complexity around K8s that we cannot magically remove :-(

Yeah I'm noticing that there's some inherent trickiness here. I think that, as long as we provide some conditional instructions and prereqs (if you are running x, be sure to set up y), we are on the right track.

I think it's best to improve the accuracy and flow in small iterations like this, especially since the application itself is still in experimental development. I also think this specific application requires the readers to fill in some information themselves: they must know how to interact with Kubernetes and apply examples to their specific configurations.
Probably the best we can do is explicitly state the technical expectations we assume of the reader (though the goal is always to be as accessible as possible).

**Pull request merged**